### DISCUSSION TITLE
October CMS compatibility

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [spatie/ray#40] [General]
October CMS 1.1.x uses Laravel 6.0 as it's the LTS version of Laravel. October CMS uses Twig instead of Blade. That's why most of the Laravel packages won't work on October CMS.

I just bought Ray because I think it's a valuable product and can help a lot of developers with debugging their code. 

I figured out it would be better to use the spatie/ray because the October CMS version of Laravel 6 uses Twig. 

To get this package to work with October CMS, I needed to fork the project and adjust some dependencies. I saw the dev-dependency includes illuminate/support 8. I removed the dev-dependency illuminate/support. 

One other dependency I changed is the ramsey/uuid. Laravel 6 is using the 3.7 version.

I would really like to use Ray in my October CMS projects. Can we get this package to work with the illuminate/support 6.x version and the ramsey/uuid 3.7?

**Comment by freekmurze** [comment]
We will probably not focus on this too much, because:

- we think following LTS is a bad idea
- a new LTS version will come later this year.

I know it sounds a bit arrogant, but I mean it very friendly:  I think this is more October's, problem than it is Ray's problem.

I might change my stance on this if there are more people asking for lower requirements.

**Comment by SebastiaanKloos** [comment]
@freekmurze Thanks for the quick reply. Really appreciate your work. I just forked the spatie/ray package and changed the deps to October CMS compatible versions. All tests are passing so that's good. I will use my own fork for now

https://github.com/kloos-dev/ray/tree/ocms
